src/kernel/liveSlots.js

```diff
@@ -17,9 +17,31 @@ function build(syscall, _state, makeRoot, forVatID) {
   }
 
   function makePresence(id) {
-    return harden({
+    const presence = harden({
       [`_importID_${id}`]() {},
     });
+    /* eslint-disable no-use-before-define */
+    // Make a handler that uses the E proxy.
+    const handler = {
+      GET(o, prop) {
+        return E(o)[prop];
+      },
+      PUT(o, prop, value) {
+        return (E(o)[prop] = value);
+      },
+      POST(o, prop, args) {
+        return prop === undefined ? E(o)(...args) : E(o)[prop](...args);
```

Comment:
Yeah, I think you should flip this around. This handler should do the work that E would have done, but without calling E.